src/gnssro/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# src/gnssro/CMakeLists.txt
+#
+# Build gnssro ncepbufr to ioda NetCDF4 converter
+#
+
+if( HAVE_BUFRLIB_AND_NETCDF )
+    set(GNSSRO_SRC gnssro_bufr2ioda.f90)
+    add_executable(gnssro_bufr2ioda ${GNSSRO_SRC})
+    target_link_libraries(gnssro_bufr2ioda bufrlib::bufrlib_shared)
```

Comment:
Can we also support static libraries? That is what was build following @srherbener 's directions in the conversation tab

Reply:
It looks like in jedi-stack we are building both the shared and static bufrlib libraries, but by default when building bufrlib you only get the static library. You have to add
`-DBUILD_SHARED_LIBS=1` to the ecbuild command line when building bufrlib to get the shared library.

I see two options:
* Change bufrlib CMake config so you get both shared and static libraries by default
* Change the line marked above to something like the following so that either static or shared can be used:
~~~~~~~~
target_link_libraries(gnssro_bufr2ioda bufrlib)
~~~~~~~~

I'm leaning toward the second option (and perhaps we should do both) since that might be necessary after getting the FindBufrlib.cmake script written which handles native installations on HPC systems. (The sys admins on the HPC systems may be inclined to only build one of static or shared.)

My vote is to put the change in to allow either static or shared (target_link_libraries command marked above) bufrlib libraries before merging this PR.

Reply:
I agree it would be best if this change could be made before the PR is merged.

Reply:
I'd like to get this resolved and merged in this week. I realize that @markjolah is swamped with other high priority tasks. @markjolah, @CoryMartin-NOAA is it okay if I make the change I proposed (change the target_link_libraries command) and then merge this in. Thanks!

Reply:
Works for me